src/Symfony/Component/Notifier/Transport/Dsn.php

```diff
@@ -59,7 +60,10 @@ public static function fromString(string $dsn): self
         $path = $parsedDsn['path'] ?? null;
         parse_str($parsedDsn['query'] ?? '', $query);
 
-        return new self($parsedDsn['scheme'], $parsedDsn['host'], $user, $password, $port, $query, $path);
+        $dsnObject = new self($parsedDsn['scheme'], $parsedDsn['host'], $user, $password, $port, $query, $path);
+        $dsnObject->dsn = $dsn;
```

Comment:
Seems that this property can be non-initialized in case when Dsn object constructed through standard public constructor. This can lead to errors during `getOriginalDsn` call

Reply:
Indeed, can you submit a PR to fix that?

Reply:
of course, will do